pkgs/applications/misc/safecloset/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, darwin
+, xorg
```

Comment:
```suggestion
, libxcb
```

Reply:
I'm a bit confused, why is xorg special cased?

I found #48219 and #195721 that seems to suggest that this is just for backwards compatibility, or are we planning to move them to top level so maybe this would be better (https://github.com/NixOS/nixpkgs/pull/195721#issuecomment-1287641577)?

Reply:
Well, the main reason is that the Xorg packages are not updated in lockstep anymore, and so they are eligible to be moved to top-level.

However, 
- this migration never happened, by lack of time and because it breaks nothing (yet)
- older aliases as `x11` and `xlibsWrapper` were effectively removed, even if they were basically the same thing;
- because of dreadful splicing (or any similar dark magic), such `xorg` indirection was discouraged.

I prefer to be explicit, because `with xorg; [ . . . ]` is morally equivalent to `with pkgs; [ . . . ]` to me.

Reply:
cc @Artturin Maybe you would have some opinions on this?

I'm still a bit hesitant to switch to `libxcb` directly since it is not available as a top-level attribute and feels a bit hacky/magic-y.

Either way, I don't think it should be a blocker for this PR, but I do want to know if there is a preference and how things might go in the future, so I can style nix-init accordingly.